examples/provisioner/100-cpu-limit.yaml

```diff
@@ -9,8 +9,15 @@ spec:
   limits:
     resources:
       cpu: 100
-  provider:
-    subnetSelector:
-      karpenter.sh/discovery: "${CLUSTER_NAME}" # replace with your cluster name
-    securityGroupSelector:
-      karpenter.sh/discovery: "${CLUSTER_NAME}" # replace with your cluster name
+  providerRef:
+    name: myProvider
+---
+apiVersion: karpenter.k8s.aws/v1alpha1
+kind: AWSNodeTemplate
+metadata:
+  name: myProvider
+spec:
+  subnetSelector:
+    karpenter.sh/discovery: "${CLUSTER_NAME}" # replace with your cluster name
+  securityGroupSelector:
+    karpenter.sh/discovery: "${CLUSTER_NAME}" # replace with your cluster name
```

Comment:
I don't think we should be DRYing our examples. I think it's a better experience to get a single manifest and apply that rather than stitching it together by applying multiple YAML files off GitHub.